<!DOCTYPE html>
<html xmlns="http://www.w3.org/1999/xhtml" lang="" xml:lang="">
<head>

  <meta charset="utf-8" />
  <meta http-equiv="X-UA-Compatible" content="IE=edge" />
  <title>Chapter 2 Gene Expression Omnibus (GEO) | BioinfoHub</title>
  <meta name="description" content="A comprehensive collection of bioinformatics techniques." />
  <meta name="generator" content="bookdown 0.12 and GitBook 2.6.7" />

  <meta property="og:title" content="Chapter 2 Gene Expression Omnibus (GEO) | BioinfoHub" />
  <meta property="og:type" content="book" />
  
  
  <meta property="og:description" content="A comprehensive collection of bioinformatics techniques." />
  <meta name="github-repo" content="rli012/rli012.github.io/BioinfoHub-pages" />

  <meta name="twitter:card" content="summary" />
  <meta name="twitter:title" content="Chapter 2 Gene Expression Omnibus (GEO) | BioinfoHub" />
  
  <meta name="twitter:description" content="A comprehensive collection of bioinformatics techniques." />
  

<meta name="author" content="Ruidong Li" />



  <meta name="viewport" content="width=device-width, initial-scale=1" />
  <meta name="apple-mobile-web-app-capable" content="yes" />
  <meta name="apple-mobile-web-app-status-bar-style" content="black" />
  
  
<link rel="prev" href="visualization.html">
<link rel="next" href="differential-expression-analysis.html">
<script src="libs/jquery-2.2.3/jquery.min.js"></script>
<link href="libs/gitbook-2.6.7/css/style.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-table.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-bookdown.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-highlight.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-search.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-fontsettings.css" rel="stylesheet" />









<style type="text/css">
a.sourceLine { display: inline-block; line-height: 1.25; }
a.sourceLine { pointer-events: none; color: inherit; text-decoration: inherit; }
a.sourceLine:empty { height: 1.2em; }
.sourceCode { overflow: visible; }
code.sourceCode { white-space: pre; position: relative; }
pre.sourceCode { margin: 0; }
@media screen {
div.sourceCode { overflow: auto; }
}
@media print {
code.sourceCode { white-space: pre-wrap; }
a.sourceLine { text-indent: -1em; padding-left: 1em; }
}
pre.numberSource a.sourceLine
  { position: relative; left: -4em; }
pre.numberSource a.sourceLine::before
  { content: attr(data-line-number);
    position: relative; left: -1em; text-align: right; vertical-align: baseline;
    border: none; pointer-events: all; display: inline-block;
    -webkit-touch-callout: none; -webkit-user-select: none;
    -khtml-user-select: none; -moz-user-select: none;
    -ms-user-select: none; user-select: none;
    padding: 0 4px; width: 4em;
    color: #aaaaaa;
  }
pre.numberSource { margin-left: 3em; border-left: 1px solid #aaaaaa;  padding-left: 4px; }
div.sourceCode
  {  }
@media screen {
a.sourceLine::before { text-decoration: underline; }
}
code span.al { color: #ff0000; font-weight: bold; } /* Alert */
code span.an { color: #60a0b0; font-weight: bold; font-style: italic; } /* Annotation */
code span.at { color: #7d9029; } /* Attribute */
code span.bn { color: #40a070; } /* BaseN */
code span.bu { } /* BuiltIn */
code span.cf { color: #007020; font-weight: bold; } /* ControlFlow */
code span.ch { color: #4070a0; } /* Char */
code span.cn { color: #880000; } /* Constant */
code span.co { color: #60a0b0; font-style: italic; } /* Comment */
code span.cv { color: #60a0b0; font-weight: bold; font-style: italic; } /* CommentVar */
code span.do { color: #ba2121; font-style: italic; } /* Documentation */
code span.dt { color: #902000; } /* DataType */
code span.dv { color: #40a070; } /* DecVal */
code span.er { color: #ff0000; font-weight: bold; } /* Error */
code span.ex { } /* Extension */
code span.fl { color: #40a070; } /* Float */
code span.fu { color: #06287e; } /* Function */
code span.im { } /* Import */
code span.in { color: #60a0b0; font-weight: bold; font-style: italic; } /* Information */
code span.kw { color: #007020; font-weight: bold; } /* Keyword */
code span.op { color: #666666; } /* Operator */
code span.ot { color: #007020; } /* Other */
code span.pp { color: #bc7a00; } /* Preprocessor */
code span.sc { color: #4070a0; } /* SpecialChar */
code span.ss { color: #bb6688; } /* SpecialString */
code span.st { color: #4070a0; } /* String */
code span.va { color: #19177c; } /* Variable */
code span.vs { color: #4070a0; } /* VerbatimString */
code span.wa { color: #60a0b0; font-weight: bold; font-style: italic; } /* Warning */
</style>

<link rel="stylesheet" href="style.css" type="text/css" />
</head>

<body>



  <div class="book without-animation with-summary font-size-2 font-family-1" data-basepath=".">

    <div class="book-summary">
      <nav role="navigation">

<ul class="summary">
<li><a href="./">BioinfoHub</a></li>

<li class="divider"></li>
<li class="chapter" data-level="" data-path="index.html"><a href="index.html"><i class="fa fa-check"></i>Introduction and Motivation</a></li>
<li class="chapter" data-level="1" data-path="visualization.html"><a href="visualization.html"><i class="fa fa-check"></i><b>1</b> Visualization</a><ul>
<li class="chapter" data-level="1.1" data-path="visualization.html"><a href="visualization.html#boxplot"><i class="fa fa-check"></i><b>1.1</b> Boxplot</a></li>
<li class="chapter" data-level="1.2" data-path="visualization.html"><a href="visualization.html#barplot"><i class="fa fa-check"></i><b>1.2</b> Barplot</a></li>
<li class="chapter" data-level="1.3" data-path="visualization.html"><a href="visualization.html#scatter-plot"><i class="fa fa-check"></i><b>1.3</b> Scatter plot</a></li>
<li class="chapter" data-level="1.4" data-path="visualization.html"><a href="visualization.html#volcano-plot"><i class="fa fa-check"></i><b>1.4</b> Volcano plot</a></li>
<li class="chapter" data-level="1.5" data-path="visualization.html"><a href="visualization.html#venn-diagram"><i class="fa fa-check"></i><b>1.5</b> Venn Diagram</a></li>
<li class="chapter" data-level="1.6" data-path="visualization.html"><a href="visualization.html#upset-plot"><i class="fa fa-check"></i><b>1.6</b> UpSet plot</a></li>
<li class="chapter" data-level="1.7" data-path="visualization.html"><a href="visualization.html#heatmap"><i class="fa fa-check"></i><b>1.7</b> Heatmap</a></li>
<li class="chapter" data-level="1.8" data-path="visualization.html"><a href="visualization.html#bubble-plot"><i class="fa fa-check"></i><b>1.8</b> Bubble plot</a></li>
<li class="chapter" data-level="1.9" data-path="visualization.html"><a href="visualization.html#radar-plot"><i class="fa fa-check"></i><b>1.9</b> Radar plot</a></li>
<li class="chapter" data-level="1.10" data-path="visualization.html"><a href="visualization.html#more"><i class="fa fa-check"></i><b>1.10</b> More â€¦</a><ul>
<li class="chapter" data-level="1.10.1" data-path="visualization.html"><a href="visualization.html#crossover-hapi"><i class="fa fa-check"></i><b>1.10.1</b> Crossover (Hapi)</a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="2" data-path="gene-expression-omnibus-geo.html"><a href="gene-expression-omnibus-geo.html"><i class="fa fa-check"></i><b>2</b> Gene Expression Omnibus (GEO)</a><ul>
<li class="chapter" data-level="2.1" data-path="gene-expression-omnibus-geo.html"><a href="gene-expression-omnibus-geo.html#affymetrix"><i class="fa fa-check"></i><b>2.1</b> Affymetrix</a></li>
</ul></li>
<li class="chapter" data-level="3" data-path="differential-expression-analysis.html"><a href="differential-expression-analysis.html"><i class="fa fa-check"></i><b>3</b> Differential Expression Analysis</a></li>
<li class="chapter" data-level="4" data-path="pathway-enrichment-anlaysis.html"><a href="pathway-enrichment-anlaysis.html"><i class="fa fa-check"></i><b>4</b> Pathway Enrichment Anlaysis</a></li>
<li class="chapter" data-level="5" data-path="single-cell-sequencing.html"><a href="single-cell-sequencing.html"><i class="fa fa-check"></i><b>5</b> Single-cell Sequencing</a></li>
<li class="chapter" data-level="6" data-path="databases.html"><a href="databases.html"><i class="fa fa-check"></i><b>6</b> Databases</a><ul>
<li class="chapter" data-level="6.1" data-path="databases.html"><a href="databases.html#tcga"><i class="fa fa-check"></i><b>6.1</b> TCGA</a></li>
<li class="chapter" data-level="6.2" data-path="databases.html"><a href="databases.html#gtex"><i class="fa fa-check"></i><b>6.2</b> GTEx</a></li>
<li class="chapter" data-level="6.3" data-path="databases.html"><a href="databases.html#ccle"><i class="fa fa-check"></i><b>6.3</b> CCLE</a></li>
<li class="chapter" data-level="6.4" data-path="databases.html"><a href="databases.html#blueprint"><i class="fa fa-check"></i><b>6.4</b> Blueprint</a></li>
<li class="chapter" data-level="6.5" data-path="databases.html"><a href="databases.html#encode"><i class="fa fa-check"></i><b>6.5</b> Encode</a></li>
<li class="chapter" data-level="6.6" data-path="databases.html"><a href="databases.html#uniprot"><i class="fa fa-check"></i><b>6.6</b> Uniprot</a></li>
<li class="chapter" data-level="6.7" data-path="databases.html"><a href="databases.html#human-protein-atlas"><i class="fa fa-check"></i><b>6.7</b> Human Protein Atlas</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="references.html"><a href="references.html"><i class="fa fa-check"></i>References</a></li>
<li class="divider"></li>
<li><a href="https://github.com/rstudio/bookdown" target="blank">Published with bookdown</a></li>

</ul>

      </nav>
    </div>

    <div class="book-body">
      <div class="body-inner">
        <div class="book-header" role="navigation">
          <h1>
            <i class="fa fa-circle-o-notch fa-spin"></i><a href="./">BioinfoHub</a>
          </h1>
        </div>

        <div class="page-wrapper" tabindex="-1" role="main">
          <div class="page-inner">

            <section class="normal" id="section-">
<div id="gene-expression-omnibus-geo" class="section level1">
<h1><span class="header-section-number">Chapter 2</span> Gene Expression Omnibus (GEO)</h1>
<p>Download and preprocess transcriptomic data from GEO</p>
<div id="affymetrix" class="section level2">
<h2><span class="header-section-number">2.1</span> Affymetrix</h2>
<div class="sourceCode" id="cb1"><pre class="sourceCode r"><code class="sourceCode r"><a class="sourceLine" id="cb1-1" data-line-number="1"><span class="kw">library</span>(oligo) <span class="co"># list.celfiles</span></a>
<a class="sourceLine" id="cb1-2" data-line-number="2"><span class="kw">library</span>(GEOquery)</a>
<a class="sourceLine" id="cb1-3" data-line-number="3"><span class="kw">library</span>(stringr) <span class="co"># str_match</span></a>
<a class="sourceLine" id="cb1-4" data-line-number="4"><span class="kw">library</span>(dplyr)</a></code></pre></div>
<div class="sourceCode" id="cb2"><pre class="sourceCode r"><code class="sourceCode r"><a class="sourceLine" id="cb2-1" data-line-number="1"><span class="co">### Annotation (GV Version 20.0.0)</span></a>
<a class="sourceLine" id="cb2-2" data-line-number="2"><span class="co"># Human Genome U133 Plus 2.0 Assay: pd.hg.u133.plus.2 [Manufacturer, Biocondutor]</span></a>
<a class="sourceLine" id="cb2-3" data-line-number="3"><span class="co"># Human Gene 1.0 ST Assay: pd.hugene10st.hs.ensg [CDF]</span></a>
<a class="sourceLine" id="cb2-4" data-line-number="4"><span class="co"># Human Gene 2.0 ST Assay: pd.hugene20st.hs.ensg [CDF]</span></a>
<a class="sourceLine" id="cb2-5" data-line-number="5"></a>
<a class="sourceLine" id="cb2-6" data-line-number="6"><span class="co"># IN USE</span></a>
<a class="sourceLine" id="cb2-7" data-line-number="7"><span class="co">#BiocManager::install(&#39;pd.hg.u133.plus.2&#39;)</span></a>
<a class="sourceLine" id="cb2-8" data-line-number="8"><span class="co">#install.packages(&quot;http://mbni.org/customcdf/20.0.0/ensg.download/pd.hugene10st.hs.ensg_20.0.0.tar.gz&quot;,</span></a>
<a class="sourceLine" id="cb2-9" data-line-number="9"><span class="co">#                 repos = NULL, type = &quot;source&quot;)</span></a>
<a class="sourceLine" id="cb2-10" data-line-number="10"><span class="co">#install.packages(&quot;http://mbni.org/customcdf/20.0.0/ensg.download/pd.hugene20st.hs.ensg_20.0.0.tar.gz&quot;,</span></a>
<a class="sourceLine" id="cb2-11" data-line-number="11"><span class="co">#                 repos = NULL, type = &quot;source&quot;)</span></a>
<a class="sourceLine" id="cb2-12" data-line-number="12"></a>
<a class="sourceLine" id="cb2-13" data-line-number="13"></a>
<a class="sourceLine" id="cb2-14" data-line-number="14"><span class="co">### Annotation (Latest Version 23.0.0, Oct 26, 2018)</span></a>
<a class="sourceLine" id="cb2-15" data-line-number="15"><span class="co"># Ensembl</span></a>
<a class="sourceLine" id="cb2-16" data-line-number="16"><span class="co">#install.packages(&quot;http://mbni.org/customcdf/23.0.0/ensg.download/pd.hgu133plus2.hs.ensg_23.0.0.tar.gz&quot;,</span></a>
<a class="sourceLine" id="cb2-17" data-line-number="17"><span class="co">#                 repos = NULL, type = &quot;source&quot;)</span></a>
<a class="sourceLine" id="cb2-18" data-line-number="18"></a>
<a class="sourceLine" id="cb2-19" data-line-number="19"><span class="co"># Gencode (Ensembl ID with version, not recommended)</span></a>
<a class="sourceLine" id="cb2-20" data-line-number="20"><span class="co">#install.packages(&quot;http://mbni.org/customcdf/23.0.0/gencodeg.download/pd.hgu133plus2.hs.gencodeg_23.0.0.tar.gz&quot;,</span></a>
<a class="sourceLine" id="cb2-21" data-line-number="21"><span class="co">#                 repos = NULL, type = &quot;source&quot;)</span></a>
<a class="sourceLine" id="cb2-22" data-line-number="22"></a>
<a class="sourceLine" id="cb2-23" data-line-number="23"><span class="kw">library</span>(pd.hg.u133.plus<span class="fl">.2</span>)</a>
<a class="sourceLine" id="cb2-24" data-line-number="24"><span class="co">#library(pd.hgu133plus2.hs.gencodeg)</span></a>
<a class="sourceLine" id="cb2-25" data-line-number="25"><span class="co">#library(pd.hgu133plus2.hs.ensg)</span></a>
<a class="sourceLine" id="cb2-26" data-line-number="26"></a>
<a class="sourceLine" id="cb2-27" data-line-number="27">pd.hg.u133.plus<span class="fl">.2</span><span class="op">@</span>tableInfo</a></code></pre></div>
<pre><code>##          tbl row_count
## 1 featureSet     54675
## 2  mmfeature    604258
## 3  pmfeature    604258</code></pre>
<div class="sourceCode" id="cb4"><pre class="sourceCode r"><code class="sourceCode r"><a class="sourceLine" id="cb4-1" data-line-number="1"><span class="co">#pd.hgu133plus2.hs.gencodeg@tableInfo</span></a>
<a class="sourceLine" id="cb4-2" data-line-number="2"><span class="co">#pd.hgu133plus2.hs.ensg@tableInfo</span></a></code></pre></div>
<div class="sourceCode" id="cb5"><pre class="sourceCode r"><code class="sourceCode r"><a class="sourceLine" id="cb5-1" data-line-number="1">gse &lt;-<span class="st"> &#39;GSE9452&#39;</span></a>
<a class="sourceLine" id="cb5-2" data-line-number="2"></a>
<a class="sourceLine" id="cb5-3" data-line-number="3"><span class="co">### Expression Data</span></a>
<a class="sourceLine" id="cb5-4" data-line-number="4">celFiles =<span class="st"> </span><span class="kw">list.celfiles</span>(<span class="kw">paste0</span>(<span class="st">&#39;data/fromGEO/&#39;</span>, gse, <span class="st">&#39;_RAW&#39;</span>), <span class="dt">full.names=</span>T, <span class="dt">listGzipped=</span>T)</a>
<a class="sourceLine" id="cb5-5" data-line-number="5">celFiles</a>
<a class="sourceLine" id="cb5-6" data-line-number="6"></a>
<a class="sourceLine" id="cb5-7" data-line-number="7">rawData =<span class="st"> </span><span class="kw">read.celfiles</span>(celFiles, <span class="dt">pkgname =</span> <span class="st">&#39;pd.hg.u133.plus.2&#39;</span>)</a>
<a class="sourceLine" id="cb5-8" data-line-number="8"><span class="co">#rawData@annotation</span></a>
<a class="sourceLine" id="cb5-9" data-line-number="9"></a>
<a class="sourceLine" id="cb5-10" data-line-number="10">probesetData =<span class="st"> </span>oligo<span class="op">::</span><span class="kw">rma</span>(rawData)</a>
<a class="sourceLine" id="cb5-11" data-line-number="11"><span class="co">#probesetData</span></a>
<a class="sourceLine" id="cb5-12" data-line-number="12"><span class="co">#probesetData@featureData@data</span></a>
<a class="sourceLine" id="cb5-13" data-line-number="13"></a>
<a class="sourceLine" id="cb5-14" data-line-number="14">exprData =<span class="st"> </span><span class="kw">exprs</span>(probesetData)</a>
<a class="sourceLine" id="cb5-15" data-line-number="15"><span class="kw">colnames</span>(exprData) &lt;-<span class="st"> </span><span class="kw">unlist</span>(<span class="kw">lapply</span>(<span class="kw">colnames</span>(exprData), <span class="cf">function</span>(x) <span class="kw">strsplit</span>(x, <span class="st">&#39;.&#39;</span>, <span class="dt">fixed=</span>T)[[<span class="dv">1</span>]][<span class="dv">1</span>]))</a>
<a class="sourceLine" id="cb5-16" data-line-number="16"></a>
<a class="sourceLine" id="cb5-17" data-line-number="17"><span class="co">#saveRDS(exprData, file=paste0(&#39;data/rData/&#39;, gse, &#39;_Expr.rds&#39;))</span></a></code></pre></div>
<div class="sourceCode" id="cb6"><pre class="sourceCode r"><code class="sourceCode r"><a class="sourceLine" id="cb6-1" data-line-number="1"><span class="co">### Phenotype Data</span></a>
<a class="sourceLine" id="cb6-2" data-line-number="2">seriesMatrix &lt;-<span class="st"> </span><span class="kw">getGEO</span>(gse, <span class="dt">AnnotGPL =</span> <span class="ot">TRUE</span>, <span class="dt">GSEMatrix =</span> <span class="ot">TRUE</span>, <span class="dt">destdir =</span> <span class="st">&#39;data/fromGEO/&#39;</span>) <span class="co"># AnnotGPL = TRUE</span></a>
<a class="sourceLine" id="cb6-3" data-line-number="3"></a>
<a class="sourceLine" id="cb6-4" data-line-number="4"><span class="co">#seriesMatrix &lt;- getGEO(filename=paste0(&#39;data/fromGEO/&#39;, gse, &#39;_series_matrix.txt.gz&#39;), AnnotGPL = FALSE) # AnnotGPL = TRUE</span></a>
<a class="sourceLine" id="cb6-5" data-line-number="5"></a>
<a class="sourceLine" id="cb6-6" data-line-number="6"><span class="co">#exprData2 &lt;- exprs(seriesMatrix[[1]])</span></a>
<a class="sourceLine" id="cb6-7" data-line-number="7"><span class="co">#exprData2[1:5,1:5]</span></a>
<a class="sourceLine" id="cb6-8" data-line-number="8"></a>
<a class="sourceLine" id="cb6-9" data-line-number="9">phenoData &lt;-<span class="st"> </span><span class="kw">pData</span>(seriesMatrix[[<span class="dv">1</span>]])</a>
<a class="sourceLine" id="cb6-10" data-line-number="10"></a>
<a class="sourceLine" id="cb6-11" data-line-number="11"><span class="co">###</span></a>
<a class="sourceLine" id="cb6-12" data-line-number="12">disease &lt;-<span class="st"> </span><span class="kw">as.character</span>(phenoData<span class="op">$</span>characteristics_ch1)</a>
<a class="sourceLine" id="cb6-13" data-line-number="13">disease[<span class="kw">grepl</span>(<span class="st">&#39;Control&#39;</span>, disease, <span class="dt">fixed=</span>T)] &lt;-<span class="st"> &#39;Healthy control&#39;</span></a>
<a class="sourceLine" id="cb6-14" data-line-number="14"></a>
<a class="sourceLine" id="cb6-15" data-line-number="15">disease[<span class="kw">grepl</span>(<span class="st">&#39;no macroscopic signs of inflammation&#39;</span>, disease, <span class="dt">fixed=</span>T)] &lt;-<span class="st"> &#39;UC non-inflamed&#39;</span></a>
<a class="sourceLine" id="cb6-16" data-line-number="16">disease[<span class="kw">grepl</span>(<span class="st">&#39;macroscopic inflammation vissible&#39;</span>, disease, <span class="dt">fixed=</span>T)] &lt;-<span class="st"> &#39;UC inflammation&#39;</span></a>
<a class="sourceLine" id="cb6-17" data-line-number="17"></a>
<a class="sourceLine" id="cb6-18" data-line-number="18">tissue &lt;-<span class="st"> &#39;Colon&#39;</span></a>
<a class="sourceLine" id="cb6-19" data-line-number="19"></a>
<a class="sourceLine" id="cb6-20" data-line-number="20">subject &lt;-<span class="st"> </span><span class="kw">str_match</span>(phenoData<span class="op">$</span>title, <span class="st">&#39;</span><span class="ch">\\</span><span class="st">d+&#39;</span>)</a>
<a class="sourceLine" id="cb6-21" data-line-number="21"></a>
<a class="sourceLine" id="cb6-22" data-line-number="22">phenoData &lt;-<span class="st"> </span><span class="kw">data.frame</span>(disease,</a>
<a class="sourceLine" id="cb6-23" data-line-number="23">                        tissue,</a>
<a class="sourceLine" id="cb6-24" data-line-number="24">                        subject,</a>
<a class="sourceLine" id="cb6-25" data-line-number="25">                        <span class="dt">row.names=</span><span class="kw">rownames</span>(phenoData))</a></code></pre></div>

</div>
</div>
            </section>

          </div>
        </div>
      </div>
<a href="visualization.html" class="navigation navigation-prev " aria-label="Previous page"><i class="fa fa-angle-left"></i></a>
<a href="differential-expression-analysis.html" class="navigation navigation-next " aria-label="Next page"><i class="fa fa-angle-right"></i></a>
    </div>
  </div>
<script src="libs/gitbook-2.6.7/js/app.min.js"></script>
<script src="libs/gitbook-2.6.7/js/lunr.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-search.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-sharing.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-fontsettings.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-bookdown.js"></script>
<script src="libs/gitbook-2.6.7/js/jquery.highlight.js"></script>
<script>
gitbook.require(["gitbook"], function(gitbook) {
gitbook.start({
"sharing": {
"github": false,
"facebook": true,
"twitter": true,
"google": false,
"linkedin": false,
"weibo": false,
"instapaper": false,
"vk": false,
"all": ["facebook", "google", "twitter", "linkedin", "weibo", "instapaper"]
},
"fontsettings": {
"theme": "white",
"family": "sans",
"size": 2
},
"edit": {
"link": "https://github.com/rstudio/bookdown-demo/edit/master/02-GEO.Rmd",
"text": "Edit"
},
"history": {
"link": null,
"text": null
},
"download": ["BioinfoHub.pdf", "BioinfoHub.epub"],
"toc": {
"collapse": "subsection"
}
});
});
</script>

</body>

</html>
